Inc/HALALMock/Services/SharedMemory/SharedMemory.hpp

```diff
@@ -9,7 +10,7 @@ class SharedMemory {
     constexpr static uint8_t max_pin_size_bytes{16};
```

Comment:
remove this variable altogether or make it initialize to `sizeof(PinModel)`. This size is already encoded in `PinModel`